docs/source/release_notes.rst

```diff
@@ -26,6 +26,7 @@ Release Notes
         * Store column typing information in a ``ColumnSchema`` object instead of a dictionary (:pr:`791`)
     * Documentation Changes
         * Update Pygments version requirement (:pr:`751`)
+        * Move spark config to a visible cell (:pr:`801`)
```

Comment:
What if we updated this line to say `Update spark config for docs build` and mention this pr and 787 from below, and then update the testing changes entry to just say `Update spark config for test fixtures` to separate these out better?

Reply:
that makes sense to me!